packages/table/src/table.tsx

```diff
@@ -95,7 +95,7 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
      * index. Note that if you want to update these values when the user
      * drag-resizes a column, you may define a callback for `onColumnWidthChanged`.
      */
-    columnWidths?: number[];
+    columnWidths?: Array<number | null | undefined>;
```

Comment:
FWIW, as the filer of the issue in question, I think accepting `null` is the right thing to do (the Robustness Principle and all that). In my experience, opinions are split in the community on whether `null`, `undefined` or both are acceptable as the "null type", and purposely not accepting one of them as a library author is poor devex. If people want to lint out their usages of `null`, that's their prerogative.